pkg/kubelet/config/common.go

```diff
@@ -43,13 +43,15 @@ func generatePodName(name string, nodeName types.NodeName) string {
 func applyDefaults(pod *api.Pod, source string, isFile bool, nodeName types.NodeName) error {
 	if len(pod.UID) == 0 {
 		hasher := md5.New()
+		hash.DeepHashObject(hasher, pod)
```

Comment:
This will trigger all static pods to be restarted during in place upgrade, right?